src/Strategy.sol

```diff
@@ -424,19 +453,18 @@ contract Strategy is BaseStrategy {
         percentKeep = _percentKeep;
     }
 
-
     // ----------------- SUPPORT & UTILITY FUNCTIONS ----------
 
     function balanceOfWant() public view returns (uint256) {
         return want.balanceOf(address(this));
     }
 
     function balanceOfStakedSanToken() public view returns (uint256) {
-        return IERC20(address(sanTokenGauge)).balanceOf(address(this));
+        return strategyProxy.balanceOf(address(sanTokenGauge));
```

Comment:
Maybe we could add a helper function like you did with balanceOfSanToken?

Reply:
I was saying add a balanceOfStakedSanToken function in the proxy, like this: https://github.com/Mattdwest/angle_protocol/blob/d7571d97f113960cdd0fbb66293acb764d764be6/src/AngleStrategyVoterProxy.sol#L167-L169